rpc-server/src/metrics.rs

```diff
@@ -166,6 +166,10 @@ lazy_static! {
         "receipt_requests_counter",
         "Total number requests to the receipt endpoint"
     ).unwrap();
+    pub static ref RECEIPT_RECORD_REQUESTS_TOTAL: IntCounter = try_create_int_counter(
```

Comment:
We have migrated to the common total counter with labels, I believe @kobayurii made a middleware for it.

I will let him to elaborate on that